atflatcontrols/atscrollbar.pas

```diff
@@ -501,12 +588,35 @@ procedure TATScrollbar.Resize;
 end;
 
 
-{$ifdef windows}
+{$ifndef FPC}
 //needed to remove flickering on resize and mouse-over
 procedure TATScrollbar.WMEraseBkgnd(var Message: TMessage);
 begin
   Message.Result:= 1;
 end;
+
+procedure TATScrollbar.CMMouseEnter(var msg: TMessage);
```

Comment:
ifdef windows - above, ifdef delphi - here

Reply:
So change to {$ifdef windows}? Or both {$ifdef windows} and {$ifdef delphi}?

Reply:
WmErase must be in ifdef windows , other Delphi stuff- in ifdef Delphi

Reply:
It means /uses Windows/ must be for FPC in ifdef Windows

Reply:
This only seems to work with {$ifndef FPC} conditional...

Reply:
Nope, it worked in FPC , with proper /uses/ which must be ok

Reply:
I have this set like this:

{$ifdef windows}
procedure TATScrollbar.WMEraseBkgnd(var Message: TMessage);
begin
  Message.Result:= 1;
end;
{$endif}

It compiles both IDEs. Do I need to change uses?

Reply:
no, uses is ok.